packages/framework/build.xml

```
@@ -750,6 +751,20 @@
 
     <target name="standards-fix-diff" depends="ecs-fix-diff,annotations-fix,yaml-standards-fix,eslint-fix-diff" description="Automatically fixes *some* coding standards violations in changed files. Always run 'standards' to be sure there are no further violations."/>
 
+    <target name="stylelint" description="Run style linter.">
```

```suggestion
    <target name="stylelint" description="Checks conventions in styles">
```

I probably suggest this change in a different file in a first CR

---

Yes, sorry, I fixed now again.